lms/djangoapps/branding/views.py

```diff
@@ -12,6 +13,8 @@
 from microsite_configuration import microsite
 from edxmako.shortcuts import marketing_link
 from util.cache import cache_if_anonymous
+from .api import get_footer_json, get_footer_static, get_footer_html
+from util.json_request import JsonResponse, HttpResponse
```

Comment:
According to [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports), imports should be grouped in the following order: standard library imports, related third party imports, and finally local application/library specific imports. There should be a blank line between each group of imports.

You can take this opportunity to clean up the way imports are arranged in this module. Your imports from `branding.models` and `brandig.api` are local and application-specific, so they should be in a third block of imports along with other local, application specific imports.

```python
from django.conf import settings
from django.core.urlresolvers import reverse
from django_future.csrf import ensure_csrf_cookie
from django.http import Http404
from django.shortcuts import redirect
 
from branding.api import get_footer_json, get_footer_static, get_footer_html
from branding.models import BrandingApiConfig
import courseware.views
from edxmako.shortcuts import marketing_link
from microsite_configuration import microsite
from student.models import CourseEnrollment
import student.views
from util.cache import cache_if_anonymous
from util.json_request import JsonResponse, HttpResponse
```